values/keycloak/keycloak.gotmpl

```diff
@@ -3,10 +3,17 @@
 {{- $consoleLoginVersion := $v.versions.consoleLogin}}
 {{- $isSemver := regexMatch "^[0-9.]+" $consoleLoginVersion }}
 
-{{- if $v.otomi.linodeLkeImageRepository }}
+{{- if or $v.otomi.linodeLkeImageRepository ($k | get "storageClass" nil) (eq $v.cluster.provider "linode") }}
```

Comment:
This logic should rely on v.cluster.defaultStorageClass instead of the provider.